### ISSUE TITLE
can't play mixer streams

### ISSUE THREAD

**Issue by marrie** [streamlink/streamlink#1400]
I can't get mixer to work. I tried my username which for example is

mixer.com/ke7zum

and it says it can't find a mixer plugin. I looked and there seems to be one. I installed this thing via homebrew so hoping i don't have to find a way to side load this into my brew install.

Over all I'm loving the program.

##troubleshooting
I made sure i was broadcasting from my xbox before going to mixer.com/ke7zum I'll keep trying, I hope I can get this to work. The error I get is 

error: No plugin can handle URL: mixer.com/ke7zum

So what did I brake?

Tc and be blessed and, thanks.

ProductName:	Mac OS X
ProductVersion:	10.13.2
BuildVersion:	17C88

**Comment by gravyboat** [comment]
Not all Mixer streams are supported (https://github.com/streamlink/streamlink/issues/497) because we aren't currently supporting FTL streams and don't currently have any plans to do so. You can rerun the command with `-l debug` and `-v` for additional output, but it's most likely that it's just an FTL stream.

**Comment by marrie** [comment]
Well, I log ed on to my stream and started streaming a game  then I did

streamlink mixer.com/ke7zum best

No go. I even tried this with a friends stream at mixer.com/jessiepp and still no go.

Is There  anything I need to do to get this working? Or anything I need to set on mixer's end some how?

Take care

**Comment by marrie** [comment]
Ok, I did the suggested command, however I got the same error even when debugging. It's like the plugin never loaded when installing via homebrew. Attached is my output, however I'm not sure if github will take the script file.




If not let me know and I'll attach it via the website later.

Thanks.

**Comment by gravyboat** [comment]
@marrie I can't see anything on GitHub itself so seems like it didn't work correctly.

**Comment by marrie** [comment]
Here is the attachment via my google drive.Ok, take two.

Thanks for the fast reply. That would by the way be the same output if I were streaming as well as I got that this afternoon when on the air, and I was streaming. I verified it with a friend.

Take care and thanks.

typescript <https://drive.google.com/uc?export=download&id=1P3RyWfC_bLhaZRfuQrGeKmHK29WUbgkO>
(364 bytes)

**Comment by back-to** [comment]
@gravyboat it's not a plugin issue, `0.9.0` is just to old.

@marrie 
your version is `0.9.0`, it does not have the mixer plugin.
wait for the [**0.10.0**](https://github.com/streamlink/streamlink/issues/1399) release

---

if you doesn't want to wait, you can sideload the plugin

- https://raw.githubusercontent.com/streamlink/streamlink/master/src/streamlink/plugins/mixer.py
- https://streamlink.github.io/cli.html#sideloading-plugins
- https://streamlink.github.io/cli.html#cmdoption-plugin-dirs

---

you can see the homebrew version here
https://github.com/Homebrew/homebrew-core/blob/master/Formula/streamlink.rb#L1-L5

**Comment by gravyboat** [comment]
@back-to Good catch, I totally forgot the old version of the plugin wasn't supporting the URL change before you made that PR.

**Comment by marrie** [comment]
Thanks thanks. I'll take a  look at side loading plugins or  wait until the home brew version gets updated. When ever that is of corse.  I'm patient. This will give me practice. I'll read the docs and see if this is easy for me to do before I head into  work.

Take care and thanks again. I'll let you know if this works probably tomorrow or tonight when ever I stream again on mixer.

Take care.

**Comment by marrie** [comment]
Ok, I read the side loading plugin docs really quickly but I don't see any of the directories. I looked for where the plugins might be stored  on mac os when using homebrew  but they are hidden, and I do have my mac to show such files and or folders.  There is nothing in my ~ directory that shows streamlink not even under !/.config. Can documentation  at some point in time  be updated to reflect the mac  folders structure as that's what I am using it on. I did dl the plugin though, thanks for that. Now to figure out where to put it and test later today.



Just for reference I'm talking about the table that shows the folder structure. I don't have a folder under mac os called 
XDG_CONFIG_HOME/streamlink/plugins
I also tried a cd to 
~/.streamlinkrc
But that folder  is not there either. I get the feeling that when installing via homebrew  somethingn is different. I took a look at the homebrew page (thanks for that) but no reference to streamlink there either. That is all  I have in the way of detail at this point in time. And that's all I actually have time for at this point in the day for troubleshooting. I hope that helps for now.

Thanks so much guys.

**Comment by skulblakka** [comment]
The config folders are not being created when installing via homebrew. (I don't know if this is actually homebrew related or just the default behavior). So you will need to create them your self. 

Instead of side loading the plugin you could also install the lastest development version via pip (you might need to run ```brew install python3```) using 
```pip install -U git+https://github.com/streamlink/streamlink.git``` (just make sure you uninstall it via brew beforehand so you don't end up with two different versions installed).

**Comment by marrie** [comment]
Ok, I found the home brew dir where plugins are. I think I side loaded the plugin correctly, although I have to still test it. But the directory where the plugins are is at:

/usr/local/Cellar/streamlink/0.9.0/libexec/lib/python2.7/site-packages/streamlink/plugins/

if anyone is curious.

Let's hope  this works. The error I got this time was actually true, as I'm not streaming at all.

[cli][info] Found matching plugin mixer for URL mixer.com/ke7zum
error: No playable streams found on this URL: mixer.com/ke7zum

So this is in fact progress. I'll probably test this tomorrow.
 Now to figure out how VOD works as according to the table where plugins are listed  I can access those.
Take care all and thanks.
